### DISCUSSION TITLE
tailwindcss or....?

### DISCUSSION THREAD

**Discussion by Nathaniel7617** [community/community#123313] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i def prefer tailwindcss 100%

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @Nathaniel7617 ! 

We’ve moved your post to our General category, which is more appropriate for this type of discussion.

**Comment by MostlyKIGuess** [comment]
**Tailwind CSS vs DaisyUI vs Bootstrap:**

* **Tailwind CSS:** Utility-first approach for maximum customization, lightweight, responsive. Steeper learning curve, potentially verbose code.
* **DaisyUI:** Built on Tailwind CSS, offers pre-built components with clean class names. Leverages Tailwind's benefits while reducing verbosity. Relies on Tailwind for core functionality.
* **Bootstrap:** Established framework with pre-built components, extensive community, and built-in JavaScript. Less customization compared to Tailwind, larger bundle size, potential for generic look.


* **Tailwind:** Maximum control, lightweight.
* **DaisyUI:** Tailwind with pre-built components, easier to learn.
* **Bootstrap:** Fast prototyping, large community, less customization.


Depends upon what you want to do and how much time you have honestly.

**Comment by iqbalrivaldi28** [comment]
I've been using bootraps for 2 years and now I'm trying to adapt to tailwind for my project. I think Tailwind is quite good when integrated into several frameworks such as Angular and perhaps other frameworks. but I think for convenience and speed, bootstraps on the other hand, tailwind has more flexible customization than bootstraps.
